dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/eols/EolsTransformationPipelineBuilder.scala

```diff
@@ -20,7 +20,8 @@ object EolsTransformationPipelineBuilder extends PipelineBuilder[Args] {
   override def buildPipeline(ctx: ScioContext, args: Args): Unit = {
     val rawRecords = readRecords(ctx, args)
 
-    val eols = rawRecords.transform("EOLS data")(_.map(EolsTransformations.mapEols))
+    val eols =
+      rawRecords.transform("EOLS data")(_.map(EolsTransformations.mapEols).filter(_.isDefined))
```

Comment:
🌱  We should get a test around this logic, but not as part of this PR. I've dropped a ticket for this work [here](https://broadinstitute.atlassian.net/browse/DSPDC-1843)